setup.py

```diff
@@ -48,8 +48,7 @@
 
 python_requires = "~=3.6"
 
-install_requires = ['torch>=1.7.0', 'torchdiffeq>=0.2.0',
-                    'torchsde @ git+https://github.com/google-research/torchsde.git']
+install_requires = ['torch>=1.7.0', 'torchdiffeq>=0.2.0', 'torchsde', 'signatory', 'pytest']
```

Comment:
I'd add a minimum version number for torchsde (say 0.2.5) for completeness.

Signatory and pytest were deliberately not included as dependencies. Signatory is difficult to install automatically, and is in any case quite a heavyweight dependency for rarely-used functionality. Pytest is only used for tests and is not needed for normal operation.

Reply:
Made the suggested changes